gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -154,6 +154,41 @@ ratelimit:
   type: mongodb
   mongodb:
     uri: mongodb://${ds.mongodb.host}:${ds.mongodb.port}/${ds.mongodb.dbname}
+#  redis:
+#    host: localhost
+#    port: 6379
+#    password:
+#    # Redis Sentinel settings
+#    sentinel:
+#      master: redis-master
+#      nodes:
+#        - host: sentinel1
+#          port: 26379
+#        - host: sentinel2
+#          port: 26379
+#    # SSL settings
+#    ssl: true
+#    trustAll: false # default value is true to keep backward compatibility
+#    tlsProtocols: TLSv1.2, TLSv1.3
+#    tlsCiphers: TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384, TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384, TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384, TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
+#    alpn: true
+#    openssl: false # Used to rely on OpenSSL Engine instead of default JDK SSL Engine
+#    keystore:
+#      type: pem # Supports jks, pem, pkcs12
+#      path: ${gravitee.home}/security/keystore.jks # A path is required if certificate's type is jks or pkcs12
```

Comment:
I don't think we should give the same keystore as for http server as the purpose isn't the same at all.
On the http server side, it is used to expose the server certificate while for redis it is used as a client certificate.

I understand that a keystore can contain multiple key pairs with proper aliases but I think it will be confusing if we mix http and redis.

It's probably better to give an example that points to a dedicated redis keystore and adapt the comment to clearly indicate that it's for configuring mTLS (client certificate) with redis. WDYT?

```yaml
# # Keystore for redis mTLS (client certificate)
#    keystore:
#      type: pem # Supports jks, pem, pkcs12
#      path: ${gravitee.home}/security/redis-keystore.jks # A path is required if certificate's type is jks or pkcs12
```

Reply:
Yeah no problem, I just copied the path to give an example of what is expected. I did not mean to advise using the same as the http server ones, but I see with your comment that it can be confusing. I'll update this with a redis prefix ;)